src/api.proposed.kernelApi.d.ts

```diff
@@ -35,9 +45,9 @@ declare module './api' {
          *
          * @param code Code to be executed.
          * @param token Triggers the cancellation of the execution.
-         * @returns Async iterable of output items, that completes when the execution is complete.
+         * @returns Async iterable of outputs, that completes when the execution is complete.
          */
-        executeCode(code: string, token: CancellationToken): AsyncIterable<OutputItem[]>;
+        executeCode(code: string, token: CancellationToken): AsyncIterable<Output>;
```

Comment:
@rebornix 
Change in the return value, I'm returning the 'NotebookCellOutput' instead of the 'NotebookCellOutputItems[]'

This is because we need to also return the metadata from outputs.
Today we're only returning raw outputs.
This is the right thing to do and also fixes the issue where DW cannot access the original traceback from the error.